src/sentry/static/sentry/app/utils/discover/fields.tsx

```diff
@@ -541,7 +541,7 @@ export const TRACING_FIELDS = [
   'percentile',
   'failure_rate',
   'apdex',
-  'user_misery',
+  'count_miserable',
```

Comment:
We need to add both `count_miserable` and `user_misery` here since they're both performance based right?

Reply:
yep, you are right